documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -74,6 +74,8 @@ export async function POST({ request, platform }) {
 }
 ```
 
+> It doesn't affect how [static variables](https://kit.svelte.dev/docs/modules#$env-static-private) work. They cant still be accessed as imports from `$env/static/private`.
```

Comment:
I don't think this is correct. I don't have a Cloudflare account, so haven't tried it, but from a glance at the code I believe all of Kit's environment variable modules are available. I.e. `$env/static/private`, `$env/dynamic/private`, `$env/static/public`, and `$env/dynamic/public`.

Even if it were only static environment variables that were still to be accessed in that manner I find it odd that you refer to only `$env/static/private`. Btw, the link should simply be `modules#$env-static-private`

Reply:
I haven't checked other three, but `$env/static/private` was available. I simply created GET endpoint to return variable and it was showing up in the response.

I'll see if I have time to create a repo to test if no one else can confirm it works this way.

Reply:
I haven't checked other three, but `$env/static/private` was available. I simply created GET endpoint to return variable and it was showing up in the response.

I'll see if I have time to create a repo to test if no one else can confirm it works this way.

Reply:
I've renamed it to Bindings and updated the other text as well, so will go ahead and merge this